src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -534,7 +578,12 @@ private void getFunctionEntryPoint(CORINFO_METHOD_STRUCT_* ftn, ref CORINFO_CONS
             if (method.IsVirtual)
                 throw new NotImplementedException("getFunctionEntryPoint");
 
-            pResult = CreateConstLookupToSymbol(_compilation.NodeFactory.MethodEntrypoint(method));
+            pResult = CreateConstLookupToSymbol(_compilation.NodeFactory.MethodEntrypoint(
+                method, 
+                constrainedType: null, 
+                originalMethod: null, 
+                methodToken: default(ModuleToken), // TODO!!!!
```

Comment:
What's the todo here? Look up with the ModuleTokenResolver?

Reply:
Correct. Right now GetHelperFtnUncached doesn't include the token info and it will likely require a bit of refactoring to push it there. One way or another I'll need to resolve this change in a non-trivial manner against the available type unification change (as the method changes need to selectively propagate to External/LocalMethodImport) so I'll try to address this TODO as part of that task.